_posts/2022-12-19-tm1-project.md

```diff
@@ -0,0 +1,222 @@
+
+
+## A take on tm1project
+
+[TOC]
```

Comment:
# Table of contents

- [A take on tm1project](#a-take-on-tm1project)
- [Overview](#overview)
  - [Git in the context of TM1](#git-in-the-context-of-tm1)
  - [What is a tm1project?](#what-is-a-tm1project)
  - [Scope and limitations](#scope-and-limitations)
- [How could you use it?](#how-could-you-use-it)
  - [Sample file](#sample-file)
- [Why should you consider it?](#why-should-you-consider-it)
- [Supported tm1project components in TM1Py](#supported-tm1project-components-in-tm1py)